micropython/bluetooth/aioble/aioble/peripheral.py

```diff
@@ -153,6 +150,9 @@ async def advertise(
                 struct.pack("<H", manufacturer[0]) + manufacturer[1],
             )
 
+        if name:
```

Comment:
Please move it to just after `services`, and add a comment, something like "Services are prioritised to go in the advertising data because iOS supports filtering scan results by service only, so service must come first".

Reply:
This change has been applied.